src/index.js

```diff
@@ -1,37 +1,103 @@
-//import Forecast from '@models/Forecast.js';
-
+import { API_URL, API_KEY } from '@models/constants.js';
+import Forecast from '@models/Forecast.js';
 import 'normalize.css';
 import "@/styles/styles.scss";
-
 import axios from 'axios';
 
-window.searchForecast = () => {
-  const input = document.querySelector('.search__input');
-  const autocomplete = new google.maps.places.Autocomplete(input);
-  autocomplete.addListener("place_changed", () => {
-    const place = autocomplete.getPlace();
+class App {
+  constructor() {
+    this.elements = {
+      input: document.querySelector('.search__input'),
+      clearButton: document.querySelector('.search__input-delete')
+    };
+
+    this.forecast = new Forecast();
+
+    this.service = new google.maps.places.PlacesService(this.elements.input);
+    this.autocomplete = new google.maps.places.Autocomplete(this.elements.input);
+
+    this.searchUserGeolocation();
+    this.addListeners();
+  }
+
+  searchUserGeolocation() {
+    navigator.geolocation.getCurrentPosition(
+      place => {
+        this.successSearch(place);
+      }, () => {
+        this.elements.input.value = 'Kyiv, Ukraine';
+        this.searchFromQuery();
```

Comment:
As I understand, you want to show Kyiv forecast as default in case user doesn't allow to get his geolocation
Weatherbit API allows you to get forecast by city name:

![image](https://user-images.githubusercontent.com/31585545/141108149-2b8fea7b-e711-4036-a797-24a3667ef695.png)

This should be enough:
```js
this.createForecast({ city: 'Kyiv' })
```